go.mod

```diff
@@ -64,12 +64,12 @@ replace (
 )
 
 require (
-	github.com/AkihiroSuda/containerd-fuse-overlayfs v1.0.0
 	github.com/NYTimes/gziphandler v1.1.1 // indirect
 	github.com/bronze1man/goStrongswanVici v0.0.0-20190828090544-27d02f80ba40 // indirect
 	github.com/containerd/cgroups v0.0.0-20200710171044-318312a37340
-	github.com/containerd/containerd v1.4.1
+	github.com/containerd/containerd v1.5.0-beta.4
```

Comment:
I don't think this is going to work since we have a replace line up above: `github.com/containerd/containerd => github.com/k3s-io/containerd v1.4.4-k3s1`

Reply:
we are not moving to containerd 1.5.x until it is out of beta afaik

Reply:
This doesn't upgrade containerd to 1.5, but `go mod tidy` fills a dummy `v1.5.0-beta.4` value here.
The actual version is `replace`d to v1.4.4-k3s1.